apps/nextjs-website/src/app/[productSlug]/guides/page.tsx

```diff
@@ -24,6 +27,26 @@ export type GuidesPageProps = {
   readonly guidesSections?: GuidesSectionProps[];
 } & LayoutProps;
 
+export const generateMetadata = async (
+  { params }: ProductParams,
+  parent: ResolvingMetadata
+): Promise<Metadata> => {
+  const resolvedParent = await parent;
+  const previousTitle = getPreviousTitle(resolvedParent);
+  const { name, path, abstract } = await getGuideLists(params?.productSlug);
+
+  const title = `${previousTitle} - ${name}`;
+
+  return {
+    title,
+    openGraph: {
+      title,
+      description: abstract?.description ?? '',
+      url: path,
+    },
+  };
+};
```

Comment:
No twitter metadata here, is that correct? Also the pages `quick-start`, `overview` and `tutorials` don't have them